### DISCUSSION TITLE
function

### DISCUSSION THREAD

**Discussion by saba1996** [YetiForceCompany/YetiForceCRM#16907] [General] [answered]
I created the module, created a duplicate file in the directory, added the event_handler class and module, but I have this error:

App\Exceptions\AppException: LBL_HANDLER_NOT_FOUND in /var/www/html/app/EventHandler.php:372
Stack trace:
#0 /var/www/html/modules/Vtiger/actions/Save.php(158): App\EventHandler->triggerHandler(Array)
#1 [internal function]: Vtiger_Save_Action->preSaveValidation(Object(App\Request))
#2 /var/www/html/app/Controller/ExposeMethod.php(65): call_user_func_array(Array, Array)
#3 /var/www/html/modules/Vtiger/actions/Save.php(73): Vtiger_Save_Action->invokeExposedMethod('preSaveValidati...', Object(App\Request))
#4 /var/www/html/include/main/WebUI.php(189): Vtiger_Save_Action->process(Object(App\Request))
#5 /var/www/html/index.php(28): Vtiger_WebUI->process(Object(App\Request))

**Comment by rskrzypczak** [comment]
First of all, please don't create duplicates (https://github.com/YetiForceCompany/YetiForceCRM/discussions/16804)
If you don't share the entries in the database where you added the handler, the location of the handler file and its class and function name, no one will be able to help you here.

**Comment by rskrzypczak** [comment, marked as answer]
Rename the folder from `Handlers` to `handlers`
case-sensitive on Linux systems.

**Reply by saba1996** [reply]
> Rename the folder from `Handlers` to `handlers` case-sensitive on Linux systems.

thanks
